src/complete_test.cpp

```diff
@@ -3302,11 +3260,22 @@ int main(int argc, char **argv) {
   time_step_client.call(time_step_srv);
 
   // test restart_controller
+  supervisor_get_from_def_srv.request.name = "ANOTHER_ROBOT";
+  supervisor_get_from_def_client.call(supervisor_get_from_def_srv);
+  uint64_t another_robot_node = 0;
+  if (supervisor_get_from_def_srv.response.node != 0) {
+    ROS_INFO("Got from DEF node: %ld.", supervisor_get_from_def_srv.response.node);
+    another_robot_node = supervisor_get_from_def_srv.response.node;
+  } else
+    ROS_ERROR("Could not ANOTHER_ROBOT node from DEF.");
```

Comment:
What does this mean? It seems a word is missing here?

Reply:
It misses a `get` that I forgot in my suggestion on this line.

Reply:
Can we fix it?

Reply:
@BenjaminHug8 @lukicdarkoo anyone?

Reply:
Should be fixed in #91.